app/models/dhs1171_pdf.rb

```diff
@@ -1,17 +1,41 @@
 class Dhs1171Pdf
-  SOURCE_PDF = "DHS_1171.pdf".freeze
+  FIELDS = [
+    "applying_for_food_assistance",
+    "birth_day",
+    "birth_month",
+    "birth_year",
+    "city",
+    "county",
+    "full_name",
+    "signature",
+    "signature_date",
+    "state",
+    "street_address",
+    "zip",
+  ].freeze
 
-  def initialize(snap_application)
+  PDF_DIRECTORY = "lib/pdfs".freeze
+  SOURCE_PDF = "#{PDF_DIRECTORY}/DHS_1171.pdf".freeze
+  COVERSHEET_PDF = "#{PDF_DIRECTORY}/michigan_snap_fax_cover_letter.pdf".freeze
+
+  def initialize(snap_application:, output_filename: "test_output.pdf")
     @snap_application = snap_application
+    @working_filename = "tmp/working_#{output_filename}"
```

Comment:
I can see why you had issues with this -- adding `tmp` all over is confusing! wonering if it would be simpler to just include it as part of the file name string rather than interpolating